include/myst/kernel.h

```diff
@@ -58,6 +58,25 @@ typedef struct _myst_wanted_secrets_config
     size_t secrets_count;
 } myst_wanted_secrets_t;
 
+typedef struct _myst_strace_config
+{
+    /* Is tracing enabled or not */
+    bool trace_syscalls;
+
+    /* Should failing syscalls be traced or not*/
+    bool trace_failing;
+
+    /* Is filtering enabled or not */
+    bool filter;
+
+    /*
+        If filter is enabled, should the given syscall be traced or not.
+        Note: ausyscall --dump shows that maximum syscall value is 332 statx.
+              The largest syscall value supported by mystikos is 2019.
+    */
+    bool trace[2020];
```

Comment:
``./kernel/times.c`` defines this:

```
#define MYST_MAX_SYSCALLS 3000
```

It would be nice to have a centralized definition in ``<myst/syscall.h>`` an replace 2020 with that constant.

Reply:
Done.